Wrappers/Python/ccpi/processors/demos_ring_removal.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
```

Comment:
Move this to `CIL-Demos` and `unittest`

Reply:
I do not know how to implement a `unittest` in this case. Demo will be added in CIL-Demos once we decide where I can add the nxs files.

Reply:
You can implement a unittest by saving the expected output and checking it with the processed one at a later stage.

Files could be uploaded to the ccpi website.